covid/__init__.py

```diff
@@ -283,7 +283,7 @@ def get_status_by_country_name(self, country_name) -> dict:
             country = next(country)
         except StopIteration:
             raise ValueError(
-                f"There is country called '{country_name}', to check availabe country names use `list_countries()`"
+                f"There is no country called '{country_name}', to check available country names use `list_countries()`"
```

Comment:
thank you very much,
I was super distracted when I wrote this.
much appreciated

Reply:
Sure, no problem!